graspologic/simulations/simulations.py

```diff
@@ -795,3 +795,195 @@ def p_from_latent(X, Y=None, rescale=False, loops=True):
         P[P < 0] = 0
         P[P > 1] = 1
     return P
+
+
+def mmsbm(
+    n,
+    p,
+    alpha=None,
+    rng=np.random.default_rng(),
+    directed=False,
+    loops=False,
+    return_labels=False,
+):
+    r"""
+    Samples a graph from Mixed Membership Stochastic Block Model (MMSBM).
+
+    MMSBM produces a graph given the specified block connectivity matrix B,
+    which indicates the probability of connection between nodes based upon
+    their community membership.
+    Each node is assigned a membership vector drawn from Dirichlet distribution
+    with parameter :math:`\alpha`. Each entry of this vector indicates the probability
+    of that node pertaining to each community. Finally, each interaction between
+    nodes is drawn by assigning the community membership of each node according to
+    the probability specified in the mixed membership vector for that node.
+
+
+    Read more in the :ref:`tutorials <simulations_tutorials>`
+
+    Parameters
+    ----------
+    n: int
+        Number of vertices of the graph.
+
+    p: array-like, shape (n_communities, n_communities)
+        Probability of an edge between each of the communities, where p[i, j] indicates
+        the probability of a connection between edges in communities [i, j].
+        0 < p[i, j] < 1 for all i, j.
```

Comment:
(here and throughout) need to make everything either math of code formatted. can look in rest of library or ask in discussion channel for examples

Reply:
For sure will be done! Apologies for not catching this earlier in the review process.